app/resources/howto.html.template

```diff
@@ -0,0 +1,130 @@
+{% extends "static-base.html.template" %}
+{% load i18n %}
+
+{% block content %}
+<h1>Person Finder User’s Guide</h1>
+
+Person Finder can be used by anyone from a PC or mobile phone.
+This document shows how to provide and search for safety information in emergency situations.
+<div class="section">
+
+<h2>Provide your own safety information</h2>
+
+<p>
+1. Getting started.
+
+<p>
+Go to Google Person Finder homepage by entering the URL(<a href="http://g.co/pf">http://g.co/pf</a>)
+ in the browser and choose the disaster name relevant to you in the left side bar.
+<a href="https://google.org/personfinder/demo">Here</a> is an environment for demonstration so you can try the operation at any time.
+<p>
+<img src="png/en/home.png" width="300">
```

Comment:
You should provide "alt" attribute for each image so that blind people can read this doc, for example. There are two options:

* alt="[Screenshot of XXX]"
* alt="" (to tell the browser that it's fine to ignore images for blind users)

Probably the latter is fine for this doc because the doc makes sense without screenshots?